core/transport/src/grpc/job_master.proto

```diff
@@ -117,6 +117,14 @@ message ListAllPResponse {
   repeated int64 jobIds = 1;
 }
 
+message ListDetailedPOptions {}
```

Comment:
Instead of making another RPC, can we overload the `ListAll` by specifying some options?